deploy.sh

```diff
@@ -52,13 +62,7 @@ do
   DEPLOY_SUBDOMAIN=`echo "$DEPLOY_SUBDOMAIN_UNFORMATTED" | sed -r 's/[^A-Za-z0-9]+/\-/g'`
   echo $DEPLOY_SUBDOMAIN
 
-  if [ "$NODE_ENV" == "production" ]
-  then
-    DEPLOY_DOMAIN=https://${DEPLOY_SUBDOMAIN}-${REPO_NAME}-${REPO_OWNER}.surge.sh
-  elif [ "$NODE_ENV" == "staging" ]
-  then
-    DEPLOY_DOMAIN=https://staging-${DEPLOY_SUBDOMAIN}-${REPO_NAME}-${REPO_OWNER}.surge.sh
-  fi
+  DEPLOY_DOMAIN=https://${DEPLOY_SUBDOMAIN}-${REPO_NAME}-${REPO_OWNER}.surge.sh
```

Comment:
btw do we really need ${REPO_NAME} or ${REPO_OWNER}? maybe we could reduce complexity of the link name by removing one of them

Reply:
@rfreitas put them in so that the script would be general to any repository. Feel free to have the debate, but I think it's out of scope of this PR.

Reply:
yeah, the fear is if someone forks this repo, they can easily get collisions with us.

Reply:
Ok I see, thanks for clarification